pkg/operator/ceph/object/health.go

```diff
@@ -175,12 +172,11 @@ func (c *bucketChecker) checkObjectStoreHealth() error {
 	return nil
 }
 
-func cleanupObjectHealthCheck(s3client *S3Agent, objectStoreUID string) error {
+func cleanupObjectHealthCheck(s3client *S3Agent, objectStoreUID string) {
 	bucketToDelete := genUniqueBucketName(objectStoreUID)
 	logger.Debugf("deleting object %q from bucket %q", s3HealthCheckObjectKey, bucketToDelete)
 	_, err := s3client.DeleteObjectInBucket(bucketToDelete, s3HealthCheckObjectKey)
-
-	return err
+	logger.Errorf("failed to delete bucker. %v", err)
```

Comment:
```suggestion
	logger.Errorf("failed to delete object in bucket. %v", err)
```

Also, check for `err != nil` before logging the error.

Reply:
Done.